flink-doris-connector/src/test/java/org/apache/doris/flink/tools/cdc/CdcPostgresSyncDatabaseCase.java

```diff
@@ -61,23 +61,27 @@ public static void main(String[] args) throws Exception{
         Configuration config = Configuration.fromMap(sourceConfig);
 
         Map<String,String> sinkConfig = new HashMap<>();
-        sinkConfig.put("fenodes","10.20.30.1:8030");
+//        sinkConfig.put("fenodes","10.20.30.1:8030");
+        sinkConfig.put("fenodes","172.20.80.2:8030");
```

Comment:
Try to hide the real IP

Reply:
This is my local test environment, do I need to delete it?

Reply:
thanks,I've deleted it